ast/treemap/treemap.h

```diff
@@ -146,6 +146,7 @@ struct ReportedRubyException {
 enum class TreeMapKind {
     Map,
     Walk,
+    ConstWalk,
```

Comment:
The `arg_type` of `TreeMapKind::Map` is `ExpressionPtr` (owned pointer, not reference), so it doesn't make sense to distinguish between a `const` and non-`const` `Map`.